src/habakkuk/make_dag.py

```diff
@@ -243,9 +247,10 @@ def runner(argv):
 
     # Check that we've been passed the name of an existing file
     if not args:
+        parser.print_help()
         raise IOError("The name of a Fortran source file must be provided.")
     if not os.path.isfile(args[0]):
```

Comment:
Realised that this only checks the first argument and not all of them. Fixed this in 55e6851.